common_design.libraries.yml

```diff
@@ -38,6 +38,11 @@ cd-article:
     component:
       components/cd-article/cd-article.css: {}
 
+cd-article-list:
+  css:
+    component:
+      components/cd-article-list/cd-article-list.css: {}
+
```

Comment:
Because we are using CSS custom properties, each library needs the ponyfill set as a dependency so it loads in IE11.
See https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/common_design.libraries.yml#L70